invenio_app_rdm/config.py

```diff
@@ -430,6 +430,17 @@ def files_rest_permission_factory(obj, action):
         'schema': 'http://www.openarchives.org/OAI/2.0/oai_dc.xsd',
         'namespace': 'http://www.openarchives.org/OAI/2.0/oai_dc/',
     },
+    'datacite': {
+        'serializer': 'invenio_rdm_records.oai:datacite_etree',
+        'schema': 'http://schema.datacite.org/'
+                  'meta/nonexistant/nonexistant.xsd',
+        'namespace': 'http://datacite.org/schema/nonexistant',
+    },
+    'oai_datacite': {
+        'serializer': ('invenio_rdm_records.oai:oai_datacite_etree'),
```

Comment:
parenthesis not needed? just trying to avoid triggering the logic of reading a tuple when is not needed 😇